packages/ihe-gateway/Dockerfile

```diff
@@ -40,12 +39,26 @@ COPY config/certs/sequoia3.pem /opt/connect/appdata/.
 COPY config/certs/sequoia4.pem /opt/connect/appdata/.
 COPY config/certs/sequoia5.pem /opt/connect/appdata/.
 COPY config/certs/$KEYSTORENAME /opt/connect/appdata/.
-RUN keytool -noprompt -importkeystore -srckeystore /opt/connect/appdata/$KEYSTORENAME -srcstoretype PKCS12 -srcstorepass $KEYSTOREPASS -destkeystore /usr/lib/jvm/zulu17/lib/security/cacerts -deststorepass $STOREPASS -deststoretype JKS
-RUN keytool -noprompt  -trustcacerts -importcert -storepass $STOREPASS -alias sequoia1 -cacerts -file /opt/connect/appdata/sequoia1.pem
-RUN keytool -noprompt  -trustcacerts -importcert -storepass $STOREPASS -alias sequoia2 -cacerts -file /opt/connect/appdata/sequoia2.pem
-RUN keytool -noprompt  -trustcacerts -importcert -storepass $STOREPASS -alias sequoia3 -cacerts -file /opt/connect/appdata/sequoia3.pem
-RUN keytool -noprompt  -trustcacerts -importcert -storepass $STOREPASS -alias sequoia4 -cacerts -file /opt/connect/appdata/sequoia4.pem
-RUN keytool -noprompt  -trustcacerts -importcert -storepass $STOREPASS -alias sequoia5 -cacerts -file /opt/connect/appdata/sequoia5.pem
+RUN --mount=type=secret,id=store_pass \
+    --mount=type=secret,id=keystore_pass \
+    keytool -noprompt -importkeystore -srckeystore /opt/connect/appdata/$KEYSTORENAME -srcstoretype PKCS12 \
+    -srcstorepass "$(grep -v '^#' /run/secrets/keystore_pass)" -destkeystore /usr/lib/jvm/zulu17/lib/security/cacerts \
```

Comment:
assuming this grep works local as well? do we have to update the setup guide for devs?

Reply:
This works local too, [I updated the README](https://github.com/metriport/metriport/pull/1422/files#diff-9a98158b63910db17f89e61c3ad020070f8b6d8192b24dfb9572dcffc183cdf0R31) indicating how to run this local (it needs a .sh script).